src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machine = {
+            new Truck(),
+            new Bulldozer(),
+            new Excavator()
```

Comment:
Consider using an abstract reference for the array type to take advantage of polymorphism. For example, `Machine[] machines = new Machine[] {...};`